tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -49,3 +56,29 @@ Create Pachyderm AWS-Secret
     Wait Until Page Contains Element        //input[@data-test-id="item-filter"]
     Input Text      //input[@data-test-id="item-filter"]    pachyderm-aws-secret
     Wait Until Page Contains Element        //a[@data-test-id="pachyderm-aws-secret"]       10
+
+Create Command In Multiple Lines
+    ${command_string}=  Catenate    SEPARATOR=${\n}
+    ...     !pachctl get file images@master:liberty.png -o original_liberty.png
+    ...     from IPython.display import Image, display
+    ...     Image(filename='original_liberty.png')
+    Log     ${command_string}
+    [Return]    ${command_string}
+
+Create Pachyderm Pipeline Using JupyterLab
+    Run Cell And Check For Errors   !git clone https://github.com/Jooho/pachyderm-operator-manifests
+    Run Cell And Check For Errors   !curl -o /tmp/pachctl.tar.gz -L https://github.com/pachyderm/pachyderm/releases/download/v2.0.5/pachctl_2.0.5_linux_amd64.tar.gz && tar -xvf /tmp/pachctl.tar.gz -C /tmp && cp /tmp/pachctl_2.0.5_linux_amd64/pachctl /opt/app-root/bin/
```

Comment:
it would be better to get the pachtl version from the installed operator rather than explicitly write it (e.g., 2.0.5), because otherwise everytime they update the operator, we will need to change this manually

Reply:
Okay. So I am fetching version and have replaced the versions by variable.